service/routes.py

```diff
@@ -138,13 +138,32 @@ def create_product_in_inventory():
     product_in_inventory.deserialize(request.get_json())
     product_in_inventory.create()
     message = product_in_inventory.serialize()
-    location_url = url_for("get_product_in_inventory",
-                           product_in_inventory=product_in_inventory.product_in_inventory_id,
-                           _external=True)
+    # location_url = url_for("get_product_in_inventory", #FIXME I had to comment these three lines to get nosetests to pass for some reason, similar to like he said this wouldn't work in his video
+    #                        product_in_inventory=product_in_inventory.product_in_inventory_id,
+    #                        _external=True)
```

Comment:
thanks for fixing this! i must have accidentally uncommented this. FYI we will officially uncomment it when @stefanisandridge completes "read" and we can call GET /inventory